src/components/link/link.style.js

```diff
@@ -20,20 +18,28 @@ a {
     &:focus {
       color: ${theme.text.color};
       cursor: default;
-      background-color: ${theme.colors.focus};
+      background-color: ${theme.colors.focusedLink};
```

Comment:
Strange variable name, I'd expect `theme.colors.focusedLink` to affect text colour.